rero_ils/modules/documents/jsonschemas/documents/document_genre_form-v0.0.1.json

```diff
@@ -0,0 +1,160 @@
+{
+  "genreForm": {
+    "title": "Genres, forms",
+    "description": "(MARC 655)",
+    "type": "array",
+    "minItems": 1,
+    "items": {
+      "type": "object",
+      "title": "Genre, form",
+      "oneOf": [
+        {
+          "title": "Link to concept entity",
+          "type": "object",
+          "additionalProperties": false,
+          "propertiesOrder": [
+            "$ref"
+          ],
+          "required": [
+            "$ref"
+          ],
+          "properties": {
+            "$ref": {
+              "title": "Concept",
+              "type": "string",
+              "pattern": "^https://mef.rero.ch/api/concepts/rero/.*?$",
+              "form": {
+                "remoteTypeahead": {
+                  "type": "mef-concepts",
+                  "enableGroupField": true
+                },
+                "templateOptions": {
+                  "itemCssClass": "col-lg-12"
+                }
+              }
+            }
+          }
+        },
+        {
+          "title": "Concept entity",
+          "type": "object",
+          "additionalProperties": false,
+          "propertiesOrder": [
+            "type",
+            "term",
+            "identifiedBy",
+            "source"
+          ],
+          "required": [
+            "type",
+            "term"
+          ],
+          "properties": {
+            "source": {
+              "title": "Source",
+              "description": "Source of the subject, for instance rero.",
+              "type": "string",
+              "default": "rero",
+              "enum": [
+                "rero"
+              ],
+              "form": {
+                "Type": "selectWithSort",
+                "options": [
+                  {
+                    "label": "rero",
+                    "value": "rero"
+                  }
+                ]
+              }
+            },
+            "type": {
+              "title": "Type",
+              "type": "string",
+              "default": "bf:Topic",
+              "enum": [
+                "rero"
+              ],
+              "form": {
+                "Type": "selectWithSort",
+                "options": [
+                  {
+                    "label": "bf:Topic",
+                    "value": "bf:Topic"
+                  }
+                ]
+              }
+            },
+            "term": {
+              "title": "Term",
+              "type": "string",
+              "minLength": 1
+            },
+            "identifiedBy": {
+              "$ref": "#/identifiedBy"
+            }
+          }
+        }
+      ]
+    },
+    "form": {
+      "hide": true
+    }
+  },
+  "identifiedBy": {
+    "title": "Identifier",
+    "type": "object",
+    "additionalProperties": false,
+    "propertiesOrder": [
+      "type",
+      "source",
+      "value"
+    ],
+    "required": [
+      "value"
+    ],
+    "properties": {
+      "type": {
```

Comment:
Can you add in the enum: "bf:Local", "rero-rameau", "gnd" ?
Default value "rero-rameau"